index.html

```diff
@@ -27,6 +27,8 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <!-- Adding some new improvments to navigation! -->
+        <!--  Navs are soooo cooooool! -->
```

Comment:
The improvements in the navigation were significant. I can see the menu is now vertical instead of horizontal.

Reply:
I did notice there was a typo. Fix Hoome to Home.